lib/irb/extend-command.rb

```diff
@@ -126,12 +129,15 @@ def irb_context
       ],
       [
         :irb_break, :Break, "cmd/break",
+        [:break, SKIP_METHOD_DEFINITION],
```

Comment:
It'd be nice to avoid defining the same alias in two different places https://github.com/ruby/irb/blob/13c2484d59b9d8e6e3f81e7b4ec9c381ab41a52c/lib/irb/init.rb#L167

Reply:
👍 Found a way to get rid of it.